<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Control_Main</class>
 <widget class="QMainWindow" name="Control_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MARC (Measurement and Auto-Test for Remote Control)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Log_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>481</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>LOGGING</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPlainTextEdit" name="pText_Log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>461</width>
       <height>621</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Consol">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: black;
	background-color: #038aa1;
	border-radius: 5px;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
     </property>
     <property name="text">
      <string>CONSOL</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/consol-icon.png</normaloff>../img/consol-icon.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_Measurement">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #80d0c7, stop:1 #13547a);
	border-radius: 20px;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}
</string>
    </property>
    <property name="text">
     <string>   MEASUREMENT</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/data-collection-icon.png</normaloff>../img/data-collection-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Instrument">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 190, 11, 255), stop:1 rgba(251, 86, 7, 255));
	border-radius: 20px;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}
</string>
    </property>
    <property name="text">
     <string>   INSTRUMENT</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/equipment-icon.png</normaloff>../img/equipment-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_configuration">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #bdc3c7, stop:1 #2c3e50);
	border-radius: 20px;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}
</string>
    </property>
    <property name="text">
     <string>   CONFIGURATION</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/setting-icon.png</normaloff>../img/setting-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_swTest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #80d0c7, stop:1 #13547a);
	border-radius: 20px;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
    </property>
    <property name="text">
     <string> SW TEST CASE</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/test.png</normaloff>../img/test.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>507</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>10</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="act_configuration"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="act_swTest"/>
    <addaction name="act_measurement"/>
    <addaction name="act_instrument"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="act_info"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Window"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFile">
   <property name="text">
    <string>Config</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionResult">
   <property name="text">
    <string>Result</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="act_measurement">
   <property name="text">
    <string>Measurement</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="act_instrument">
   <property name="text">
    <string>Instrument</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="act_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="act_configuration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="act_swTest">
   <property name="text">
    <string>SwTest</string>
   </property>
  </action>
  <action name="act_info">
   <property name="text">
    <string>About SW</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
